backend/src/main/java/com/google/models/Anomaly.java

```diff
@@ -98,12 +111,14 @@ public String toString() {
     dataPoints.forEach((key, value) -> 
       str.append(key + ": " + value + "\n")
     );
+    str.append("Related Data: \n");
+    relatedDataList.forEach(data -> str.append(data));
```

Comment:
since there's already a `toString()` defined for `RelatedData` when I do `str.append(data)` it automatically calls the `toString()` function, so I don't need to call it.